docs/index.md

```diff
@@ -884,7 +884,31 @@ Options:
   
 ```
 
-## CSV Output
+### Scaling up recipe execution
+
+From the command line you can control how many rows a recipe generates by telling
+Snowfakery how many times to execute it. You do this by specifying a "target count"
```

Comment:
Based on the explanation below, "how many times to execute it" is a bit misleading.

Reply:
99d3fc3